bedrock/settings/base.py

```diff
@@ -1225,6 +1231,7 @@ def lazy_email_backend():
 CSP_IMG_SRC = CSP_DEFAULT_SRC + (
     'data:',
     '*.optimizely.com',
+    '*.statcounter.com',
```

Comment:
For ease of clean-up, lets move these CSP exceptions into a conditional like we do for `ad.doubleclick.net`. Remove `*.statcounter.com` and `www.statcounter.com` from the canonical lists and move into an `if` block at the bottom of the file (underneath the `if config('SWITCH_TRACKING_PIXEL', default=DEV, cast=bool):` block). Should look like:

```
# See https://bugzilla.mozilla.org/show_bug.cgi?id=1352821
if config('SWITCH_STATCOUNTER_EXPERIMENT', default=DEV, cast=bool):
    CSP_IMG_SRC += ('*.statcounter.com',)
    CSP_SCRIPT_SRC += ('www.statcounter.com',)
```